AppCenterReactNativeShared/ios/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
```

Comment:
Just nit-picking, I think we can get rid of these two lines of comments as we won't need it for global scope anyway.